src/OrbitGl/TimelineInfoInterface.h

```diff
@@ -9,6 +9,8 @@
 
 namespace orbit_gl {
 
+constexpr float kIncrementalZoomTimeRatio = 0.1;
+
```

Comment:
Is there a reason for this to be here? It's an implementation detail that IMO shouldn't be in the interface header. If you want this to be publicly available, I'd suggest to define it in `TimeGraph.h` instead.